mofacts/client/views/experiment/card.js

```diff
@@ -250,10 +250,10 @@ function checkSimulation() {
     return;
   }
 
-  const setspec = Session.get('currentTdfFile').tdfs.tutor.setspec[0];
+  const setspec = Session.get('currentTdfFile').tdfs.tutor.setspec;
 
-  const simTimeout = _.chain(setspec).prop('simTimeout').intval(0).value();
-  const simCorrectProb = _.chain(setspec).prop('simCorrectProb').floatval(0.0).value();
+  const simTimeout = parseInt(setspec.simTimeout);
```

Comment:
What happens if simTimeout isn't in the tdf?

Reply:
I seem to have forgotten to add the `|| 0` part to the line